tasks/main.yml

```diff
@@ -158,6 +158,12 @@
     name: "{{ item.value }}"
   with_dict: "{{ users }}"
 
+- name: add www-data to phab-daemon group
+  user:
+    name: "{{ users.web }}"
+    groups: "phab-daemon"
+    append: true
+
```

Comment:
Hi @trondstromme  .
Thanks for the pull request.
Just trying to understand more.
- Is there a link in the official documentation that recommends doing this step?   Or in a support ticket where epriestley suggests it?
- What features are broken without the change? 
- Could it be a security risk? It's good to limit the web user as much as possible.  
By adding the web to the phabricator group, would it have far reaching permissions?